src/storage/page_buffer.c

```diff
@@ -7856,28 +7859,29 @@ pgbuf_request_data_page_from_page_server (const VPID * vpid)
   /* Send a request to Page Server for the Page. */
   if (get_server_type () == SERVER_TYPE_TRANSACTION)
     {
-      constexpr size_t PAGEID_SIZE = sizeof (VPID::pageid);
-      constexpr size_t VOLID_SIZE = sizeof (VPID::volid);
-      char buffer[PAGEID_SIZE + VOLID_SIZE + sizeof (LOG_LSA)];
+      cubpacking::packer pac;
+      int size = 0;
 
-      std::memcpy (buffer, &(vpid->pageid), sizeof (vpid->pageid));
-      size_t bytes_copied = sizeof (vpid->pageid);
-
-      std::memcpy (buffer + bytes_copied, &(vpid->volid), sizeof (vpid->volid));
-      bytes_copied += sizeof (vpid->volid);
+      // *INDENT-OFF*
+      size += cublog::lsa_utils::get_packed_size(pac, size);
+      size += vpid_utils::get_packed_size(pac, size);
+      char buffer[size];
 
+      pac.set_buffer (buffer, size);
+      vpid_utils::pack(*vpid, pac);
       LOG_LSA nxio_lsa = log_Gl.append.get_nxio_lsa ();
-      std::memcpy (buffer + bytes_copied, &nxio_lsa, sizeof (nxio_lsa));
-      bytes_copied += sizeof (nxio_lsa);
+      cublog::lsa_utils::pack(nxio_lsa, pac);
+      // *INDENT-ON*
 
-      std::string message (buffer, bytes_copied);
-      ats_Gl.push_request (ats_to_ps_request::SEND_DATA_PAGE_FETCH, std::move (message));
-      if (prm_get_bool_value (PRM_ID_ER_LOG_READ_DATA_PAGE))
-	{
-	  _er_log_debug (ARG_FILE_LINE, "Sent request for Page to Page Server. pageid: %ld volid: %d\n", vpid->pageid,
-			 vpid->volid);
-	}
+  std::string message (buffer, size);
```

Comment:
is there any change here besides the indent? if not, can you please check your indent?